src/m365/aad/commands/group/group-list.ts

```diff
@@ -1,11 +1,16 @@
 import { Group } from '@microsoft/microsoft-graph-types';
 import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
 import GlobalOptions from '../../../../GlobalOptions';
 import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
 import commands from '../../commands';
 
 interface CommandArgs {
-  options: GlobalOptions;
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  deleted?: boolean;
```

Comment:
Let's add tracking the usage of this option in the telemetry

Reply:
Will do it